doc/topics/price-solver.md

```diff
@@ -0,0 +1,22 @@
+# Using Price Solver
+
+Price solver is a tool to calculate the price of a market based on the prices of other markets. It is useful when you
+want to calculate the price of a market that is not directly available on the exchange.
```

Comment:
Could you provide an example in the comment? This would be very helpful!

Reply:
Yes, in the next PR, thanks!